src/components/unique-token/UniqueTokenCard.js

```diff
@@ -25,7 +25,7 @@ const UniqueTokenCard = ({
   disabled,
   enableHapticFeedback = true,
   height,
-  item: { background, image_preview_url, ...item },
+  item: uniqueToken,
```

Comment:
why any changes here?

Reply:
We were needed to pass the `uniqueToken` in it's entirety to a rendered component. These variables are still destructed, just this is done in the function body.

Reply:
Please take a look at the overall diff for this file. Those changes are no-op

Reply:
Ah, thanks for catching this. Looks like the use case disappeared.